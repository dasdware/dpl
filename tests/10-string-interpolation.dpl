print("1 + 2 = ${1 + 2}\n");

type Person := $[ firstname: String, lastname: String ];

function toString(p: Person)
    := "${p.firstname} ${p.lastname}";

var p: Person := $[ firstname := "John", lastname := "Doe"];
print("p is ${p}\n");