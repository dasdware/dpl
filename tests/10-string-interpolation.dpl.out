1 + 2 = 3
p is John Doe
